packages/slate-editor/src/modules/editor/lib/useCursorInView.ts

```diff
@@ -29,7 +30,7 @@ function ensureCursorInView(editor: Editor, parameters: Parameters): void {
     }
     const [currentNode] = EditorCommands.getCurrentNodeEntry(editor) || [];
 
-    if (Range.isExpanded(editor.selection) && !isImageNode(currentNode)) {
+    if (Range.isExpanded(editor.selection) || isImageNode(currentNode)) {
```

Comment:
This is opposite to these changes https://github.com/prezly/slate/pull/149/files#diff-3189cee2b014dfe319eea38ebdec2dd4e9d0447af3542754a6f35aeb7cdef1aeR28 what is weird.

@OskarGasiorowski could you explain what behavior you tried to fix in that PR for images?